100-Appendix/025-sla.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Xata Service Level Agreement (SLA)
+navTitle: SLA
+keywords: ['sla', 'slo', 'service level agreement']
+description: Xata offers a Service Level Agreement (SLA) for its paid plans.
+slug: appendix/sla
+published: true
+---
+
+The following SLA applies to the Xata Pro plan. The Xata free tier doesn't include an SLA, and the Xata Enterprise plan SLA can be customized. The SLA applies separately to each workspace.
+
+## Service Layer Agreement
+
+The following Service Level Agreement (SLA) applies to the core Postgres database service, for both Shared and Dedicated Clusters. It includes access over the Postgres wire protocol as well as over the Data API.
+
+| Monthly Uptime Percentage                          | Service Credit Percentage |
+| -------------------------------------------------- | ------------------------- |
+| Less than 99.9% but equal to or greater than 99.0% | 10%                       |
+| Less than 99.0% but equal to or greater than 95.0% | 25%                       |
+| Less than 95.0%                                    | 100%                      |
+
+## Service credits
+
+Service Credits are calculated as a percentage of the charges paid by you for Xata for the monthly billing cycle in which the applicable SLA was not met. We will apply any Service Credits only against future Xata payments otherwise due from you. Service Credits may not be transferred or applied to any other account.
```

Comment:
```suggestion
Service Credits are calculated as a percentage of the charges paid by you for Xata for the monthly billing cycle in which the applicable SLA was not met. We will apply any Service Credits only against future payments otherwise due from you. Service Credits may not be transferred or applied to any other account.
```